### PR TITLE
id,zh-CN: update translations

### PR DESCRIPTION
**Pull request by ovflowd**
Update the Indonesian and the Simplified Chinese translations.

### CODE REVIEW

**Review by Mr7Gabriel — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Approved for the Simplified Chinese

File: pages/id/docs/guides/nodejs-docker-webapp.md [hunk 1]
```diff
@@ -66,10 +66,10 @@ touch Dockerfile
 
 Buka `Dockerfile` di editor teks favorit Anda
 
-Hal pertama yang perlu kita lakukan adalah menentukan dari image apa kita ingin membangun. Di sini kita akan menggunakan LTS (dukungan jangka panjang) versi `16` terbaru dari `node` tersedia dari [Docker Hub](https://hub.docker.com/_/node):
+Hal pertama yang perlu kita lakukan adalah menentukan dari gambar apa kita ingin membangun. Di sini kita akan menggunakan versi terbaru LTS (dukungan jangka panjang) `18` dari `node` tersedia dari [Docker Hub](https://hub.docker.com/_/node):
 
 ```docker
-FROM node:16
+DARI node: 18
```
Comment (HinataKah0) [inline pages/id/docs/guides/nodejs-docker-webapp.md]: This seems wrong... lol
`FROM` is a keyword...

Comment (Mr7Gabriel) [reply pages/id/docs/guides/nodejs-docker-webapp.md]: > This seems wrong... lol `FROM` is a keyword...

Initially, I also thought the same way, but Crowdin does not show it as a keyword. 👀
Thank you for the review, I'll fix it :)

Comment (AugustinMauroy) [reply pages/id/docs/guides/nodejs-docker-webapp.md]: @ovflowd 👋

you could look into crowdin's settings to disable code translation (this is what's been done on webdriver.io). Unless you don't have access to .

Comment (ovflowd) [reply pages/id/docs/guides/nodejs-docker-webapp.md]: I don't see any option for "disabling code translations", but will give an eye :)

Comment (ovflowd) [reply pages/id/docs/guides/nodejs-docker-webapp.md]: I was able to find the setting and disabled the ability to translate code blocks on both pure Markdown and MDX. I also disabled "content segmentation" or Markdown and I'm curious to see if this affects positively these MDX issues we've having?

Comment (ghost) [reply pages/id/docs/guides/nodejs-docker-webapp.md]: > Initially, I also thought the same way, but Crowdin does not show it as a keyword. 👀 Thank you for the review, I'll fix it :)

As for @HinataKah0 's pointing out：Just a suggestion：We tend to keep what the comments and commands of codes what they really are (Not translations), bcoz we will execute these commands step by step :)

**Comment by Mr7Gabriel** [comment]
In the coming days, I will conduct a thorough check on the complete translation of the ID (Indonesian) to anticipate the possibility of typos or translation errors in the code that should not have been translated by me.

I apologize for the oversight I made 😓, and I want to rectify it as soon as possible, but I have many things to attend to in the next few days.

Thank you, @ovflowd, for your update translation. I will review my translations going forward. Also, thank you to everyone else for the suggestions you have provided to me. I highly appreciate the advice you have given me to help me improve my translations.

**Comment by ovflowd** [comment]
No worries, @Mr7Gabriel :)

**Pull request merged**